lms/templates/registration/password_reset_confirm.html

```diff
@@ -36,115 +36,64 @@
 {% block bodyclass %}view-passwordreset{% endblock %}
 
 {% block body %}
-  <section class="introduction">
-    <header>
-      <h1 class="title">
-        <span class="title-super">
-          {% blocktrans with platform_name=platform_name %}
-          Reset Your {{ platform_name }} Password
-          {% endblocktrans %}
-        </span>
-      </h1>
-    </header>
-  </section>
 
-  <section class="passwordreset container">
-    <section role="main" class="content">
-      {% if validlink %}
-      <header>
-      <h2 class="sr">{% trans "Password Reset Form" %}</h2>
-      </header>
+    <div id="login-and-registration-container" class="login-register">
+        <section id="password-reset-confirm-anchor" class="form-type">
+            <div id="password-reset-confirm-form" class="form-wrapper" aria-hidden="true">
+                <div class="status submission-error {% if not err_msg %} hidden {% endif %}" aria-live="polite">
+                    <h4 class="message-title">{% trans "The following errors occurred while processing your registration: " %}</h4>
+                    <ul class="message-copy">
+                    {% if err_msg %}
```

Comment:
Ignore this. This is indeed the preferred way as seen in django code examples: https://docs.djangoproject.com/es/1.9/ref/templates/builtins/#regroup